song-server/src/main/java/bio/overture/song/server/model/analysis/Analysis.java

```diff
@@ -69,9 +61,20 @@
   @Column(name = TableAttributeNames.STATE, nullable = false)
   private String analysisState = UNPUBLISHED.name();
 
+  @Column(name = TableAttributeNames.CREATED_AT, nullable = false)
+  @CreationTimestamp
+  private LocalDateTime createdAt;
+
+  @Column(name = TableAttributeNames.UPDATED_AT, nullable = false)
+  @UpdateTimestamp
+  private LocalDateTime updatedAt;
+
+  @Transient private LocalDateTime firstPublishedAt;
+  @Transient private LocalDateTime publishedAt;
+
   @NotNull
   @JsonIgnore
-  @ManyToOne(fetch = FetchType.LAZY)
+  @ManyToOne(fetch = FetchType.EAGER)
```

Comment:
This doesn't look right and could have big performance implications. 

Why not use the `AnalysisSpecificationBuilder` at query time to dictate which lazy loaded elements you want included?

Reply:
Thats because I didn't mean to do this. Thanks for catching it.

Reply:
You might want to make your new table also lazy loaded in its join to Analysis.